app/models/cluster.rb

```diff
@@ -81,5 +87,8 @@ def save!
     prefixed_set(:instance_type, @instance_type)
     prefixed_set(:instance_type_custom, @instance_type)
     prefixed_set(:instance_count, @instance_count)
+    prefixed_set(
```

Comment:
compared to other code, this reads ugly. 
I searched a bit it seems we need to convert ths to a true/false variable.

Maybe also could be ok:

``` ruby
  boo_hana =  ActiveModel::Type::Boolean.new.cast(@hana_ha_enabled)
 prefixed_set( :hana_ha_enabled, bool_hana)
```

Reply:
This is how it is used in other parts of the code